web-console/knikubevirt/Create-vm/step-5-review/wizard-review.md

```diff
@@ -0,0 +1,36 @@
+# Create Virtual Machine Wizard - Review Step
+
+The purpose of this doc is to update the current create VM designs to align with OpenShift and Patternfly 4.
+
+Some things to note:
+
+The “Create virtual machine” action has been updated to “Review and create”.
+Once the user has clicked this button they will taken to the Review screen to verify their settings.
+
+![user clicks review and create button](img/review-and-create-action.png)
+
+The action “Create virtual machine” has been moved to the footer with the other actions.
+This encourages the user to check their settings quickly before creating the VM.
+
+![user checks settings before creating vm](img/check-settings.png)
```

Comment:
The action should be "Create virtual machine" now that we are located on the Review page. I also think we should remove the "next" button here now that our primary action is "Create virtual machine". I don't see a need for next disabled.

Reply:
Agree about changing the button on the "Review" step and removing "next"